### ISSUE TITLE
Feature request: previous and next activity buttons

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetTools#1594]
It'd be great to have a button that allows a user to go to the previous or next activity in the course outline.

**Comment by Lyanthropos** [comment]
These are available within the activity viewer to take you to the previous and next activities. Checkout #1817. Would you also want them to be available in the editor? 

What is considered the "current" in the course outline?

**Comment by dqnykamp** [comment]
I would include the buttons only when view assigned activity, not when view draft activity or edit activity, i.e., what you have right now.

I don't understand what you mean by "current".

**Comment by Lyanthropos** [comment]
Your original comment talks about 
> ... in the course outline.

I wanted to make sure this PR would cover what you were expecting

**Comment by dqnykamp** [comment]
Just like with the Gradebook ordering, I'd like to make sure they go to the next/previous item that one would see when browsing the Content (the course outline).  Also, make sure that it only includes assigned activities.

I'm not sure how to handle activities that are marked "proctor makes available".   We should presumably exclude those, since a student can't access those anyway.  Unless it would be useful to indicate to the student that this is where the exam fits in.

**Comment by Lyanthropos** [comment]
Yes, currently it will cycle through activities that show up in the Content for the given user. If you have `canViewUnassignedContent` you'll be able to see what is not yet assigned (it will say content not assigned). Switch roles to see it in effect. 

If they can see exams in the course flow, I would keep it in the cycle for parity.

**Comment by dqnykamp** [comment]
I'd like to shop around this question about whether or not the `previous` and `next` buttons should depend on role.  I hadn't thought of making it change with role.

I see your argument of maintaining parity with the Content list.  But, it's not useful to get these "content not assigned" or "available only in proctored setting" messages.

I've discovered that many students don't even look at the Content list.  (The `previous` and `next` buttons are a way to sneak some of that structure into their awareness, as well as it being a requested feature by students.)  In that case, parity with the Content list may not be an important consideration.

**Comment by dqnykamp** [comment]
From asking folks in our authoring group, there was no consensus about what the `previous` and `next` buttons should do for instructors/authors.  So, I think do whatever you want in that case, and then we'll modify it after people get a chance to see it in action.

For the students, though, I think you should skip over "proctor makes available" assignments for now.  Even though they do show up in the course outline, having the `next` button dropping you to a screen that shows nothing other than the warning "Assignment only available in a proctored setting" will be disconcerting.

We should really have something else show for students when they try to go to an assessment that has "proctor makes available"  set.  It should be an explanation page about the exam.  Once that is in place, it would make sense for the `previous` and `next` buttons to include such assignments.

I'll put in a request for having a different screen for proctored assignments.
